hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -200,6 +200,20 @@
       milestone: V2
       tests: ["flash_ctrl_error_prog_type"]
     }
+    {
+      name: read_write_overflow
+      desc: '''
+            Send following error transactions with normal traffic and see
+            any catastrophic event happens.
+              - Program flash size longer than 64 bytes
+              - Read flash from controller without settting start op
+              - Issue rd_fifo read more thatn CONTROL.NUM field value.
```

Comment:
Maybe add the end period `.` to all the end of the sentence, or remove them all?

Reply:
fixed